### DISCUSSION TITLE
Anyone plz help me

### DISCUSSION THREAD

**Discussion by Rokie084** [BepInEx/BepInEx#984] [Help and Q&A] [answered]
Can you find problem listed below?

[Message:   BepInEx] Chainloader ready
[Message:   BepInEx] Chainloader started
[Info   :   BepInEx] 20 plugins to load
[Warning:   BepInEx] Skipping [BGM Loader 20.0] because of process filters (AI-Syoujyo)
[Warning:   BepInEx] Skipping [Configuration Manager wrapper for AI-Shoujo 20.0] because of process filters (AI-Syoujyo, StudioNEOV2)
[Warning:   BepInEx] Skipping [Extended Save 20.0] because of process filters (AI-Syoujyo, StudioNEOV2)
[Warning:   BepInEx] Skipping [Input Length Unlocker 20.0] because of process filters (AI-Syoujyo, StudioNEOV2)
[Warning:   BepInEx] Skipping [Screenshot Manager 20.0] because of process filters (AI-Syoujyo, StudioNEOV2)
[Warning:   BepInEx] Skipping [Sideloader 20.0] because of process filters (AI-Syoujyo, StudioNEOV2)
[Warning:   BepInEx] Skipping [Slider Unlocker 20.0] because of process filters (AI-Syoujyo, StudioNEOV2)
[Info   :   BepInEx] Loading [BepInEx.IPALoader 1.2.4]
[Error  :   BepInEx] Could not load [Cheat Tools 2.7] because it has missing dependencies: RuntimeUnityEditor (v2.0 or newer)
[Info   :   BepInEx] Loading [Configuration Manager 18.3]
[Error  :   BepInEx] Could not load [Modding API 1.41] because it has missing dependencies: com.bepis.bepinex.extendedsave (v12.2 or newer), com.bepis.bepinex.extendedsave (v19.4 or newer)
[Warning:   BepInEx] Skipping [Skin Overlay Mod 6.1.5] because it has a dependency that was not loaded. See previous errors for details.
[Warning:   BepInEx] Skipping [Uncensor Selector 3.12.1] because it has a dependency that was not loaded. See previous errors for details.
[Info   :   BepInEx] Loading [BonesFramework 1.4.1]
[Info   :   BepInEx] Loading [XUnity Resource Redirector 2.1.0]
[Warning:   BepInEx] Skipping [Clothes Overlay Mod 6.1.5] because it has a dependency that was not loaded. See previous errors for details.
[Info   :   BepInEx] Loading [Clothes Overlay Mod GUI 6.1.5]
[Warning:   BepInEx] Skipping [AIABMX (BonemodX) 5.2.2] because it has a dependency that was not loaded. See previous errors for details.
[Info   :   BepInEx] Loading [Skin Overlay Mod GUI 6.1.5]
[Warning:   BepInEx] Skipping [Card Author Data 1.41] because it has a dependency that was not loaded. See previous errors for details.
[Message:   BepInEx] Chainloader startup complete
[Info   :   Console] Searching for compatible XInput library...
[Info   :   Console] Found Xinput1_4.dll.

**Comment by Rokie084** [comment, marked as answer]
problem solved

**Reply by wbaiyounv-hub** [reply]
How did you resolve it?
